tests/kernel/grub_baremetal.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Handle grub menu on ipmi console
+# Maintainer: Michael Moese <mmoese@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    return unless check_var('BACKEND', 'ipmi');
+
+    my $timeout = get_var('GRUB_TIMEOUT', 300);
+
+    select_console 'sol', await_console => 0;
+    assert_screen_with_soft_timeout('grub2', timeout => 2 * $timeout, soft_timeout => $timeout, bugref => 'boo#1120256');
+    send_key 'ret';
+    assert_screen('linux-login', 300);
+}
+
+sub test_flags {
+    return {fatal => 1};
+}
+
+1;
```

Comment:
Whole `grub_baremetal` seems to be redundant, as we discussed on IRC...

Reply:
I removed this completely.